### PR TITLE
Add API /studio/api/2/content/rename

### PR DESCRIPTION
**Pull request by phuongnq**
### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/5630

### CODE REVIEW

File: src/main/java/org/craftercms/studio/model/rest/content/RenameRequestBody.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.model.rest.content;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.validation.constraints.NotEmpty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (jmendeza) [inline src/main/java/org/craftercms/studio/model/rest/content/RenameRequestBody.java]: Do we want to ignore unknown invalid properties? Several tickets have been reported requiring to remove ignoreUnknown param
cc: @Euquimides

**Review by jmendeza — CHANGES_REQUESTED** [review]
Please review my comment about @JsonIgnoreProperties(ignoreUnknown = true)

**Comment by sumerjabri** [comment]
@phuongnq please see the notes

File: src/main/java/org/craftercms/studio/model/rest/content/RenameRequestBody.java (hunk 1 above)
Comment (phuongnq) [reply src/main/java/org/craftercms/studio/model/rest/content/RenameRequestBody.java]: Removed `@JsonIgnoreProperties(ignoreUnknown = true)` notation. Thank you!

**Review by jmendeza — APPROVED** [review]

**Pull request merged**